src/Common/src/System/Net/Security/Unix/SafeFreeSslCredentials.cs

```diff
@@ -79,8 +85,22 @@ public SafeFreeSslCredentials(X509Certificate certificate, SslProtocols protocol
                     throw new NotSupportedException(SR.net_ssl_io_no_server_cert);
                 }
 
-                _certHandle = Interop.Crypto.X509Duplicate(cert.Handle);
+                _certHandle = Interop.Crypto.X509Duplicate(cert.Handle);                
```

Comment:
Nit: There's a lot of whitespace at the end of this line, and a few other blank lines that turned into whitespace.